src/common/myclient.ts

```diff
@@ -44,7 +44,9 @@ export function MyClientWithBot<T_SpecificGameState,T_SpecificPosition>(
     board: boardWrapper(board, description),
     multiplayer: Local(
       {
-        bots: { '1': botWrapper(strategy) }
+        bots: { '1': botWrapper(strategy) },
+        persist: true,
+        storageKey: 'strategy_' + game.name,
```

Comment:
I think you can use the same prefix for strategy and relay, since they should have different different game names.